### PR TITLE
[MO] - [Parallel namespace test] -> KafkaST

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Type of change

- Enhancement / new feature

### Description

This PR adding support for parallel namespace execution for `KafkaST` suite.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Comment by see-quick** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST

**Comment by see-quick** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST

File: systemtest/src/test/resources/junit-platform.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # These are default values if @ParallelTest or @ParallelSuite is not specified
 # junit.jupiter.execution.parallel.mode.default <==> ParallelTest = concurrent
 # junit.jupiter.execution.parallel.mode.classes.default <==> ParallelSuite = concurrent
-junit.jupiter.execution.parallel.enabled=false
+junit.jupiter.execution.parallel.enabled=true
 junit.jupiter.execution.parallel.mode.default=same_thread
 junit.jupiter.execution.parallel.mode.classes.default=same_thread
 junit.jupiter.execution.parallel.config.strategy=fixed
-junit.jupiter.execution.parallel.config.fixed.parallelism=1
+junit.jupiter.execution.parallel.config.fixed.parallelism=5
```
Comment (Frawless) [inline systemtest/src/test/resources/junit-platform.properties:8]: leftover?

File: test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java [hunk 2]
```diff
@@ -241,6 +260,18 @@ public void deleteCustomResources(String... resources) {
         }
     }
 
+    public void deleteCustomResources(ExtensionContext extensionContext, String... resources) {
+        final String namespaceName = !extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get("NAMESPACE_NAME").toString().isEmpty() ?
+            extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get("NAMESPACE_NAME").toString() :
+            getNamespace();
+
+        for (String resource : resources) {
```
Comment (Frawless) [inline test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java:268]: What is in resources? yaml contents or paths?

Comment (see-quick) [reply test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java:268]: `@param resources array of paths to yaml files with resources specifications`

File: systemtest/src/test/resources/junit-platform.properties (hunk 1 above)
Comment (see-quick) [reply systemtest/src/test/resources/junit-platform.properties]: yes, thanks

**Comment by see-quick** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST

**Review by scholzj — APPROVED** [review]

**Comment by see-quick** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST

**Comment by see-quick** [comment]
@strimzi-ci run tests test_only profile=regression testcase=KafkaST

**Pull request merged**